torch/testing/_deprecated.py

```diff
@@ -29,3 +36,48 @@ def inner_wrapper(*args: Any, **kwargs: Any) -> Any:
 
 rand = warn_deprecated("Use torch.rand instead.")(torch.rand)
 randn = warn_deprecated("Use torch.randn instead.")(torch.randn)
+
+
+_DTYPE_PRECISIONS = {
+    torch.float16: (1e-3, 1e-3),
+    torch.float32: (1e-4, 1e-5),
+    torch.float64: (1e-5, 1e-8),
+}
+
+
+def _get_default_rtol_and_atol(actual: torch.Tensor, expected: torch.Tensor) -> Tuple[float, float]:
+    actual_rtol, actual_atol = _DTYPE_PRECISIONS.get(actual.dtype, (0.0, 0.0))
+    expected_rtol, expected_atol = _DTYPE_PRECISIONS.get(expected.dtype, (0.0, 0.0))
+    return max(actual_rtol, expected_rtol), max(actual_atol, expected_atol)
+
+
+# TODO: include better migration instructions.
+@warn_deprecated("Use torch.testing.assert_close instead.")
```

Comment:
@mruberry Instead of posting a full upgrade guide here, is it ok to link to the RFC in #61844?